app/controllers/users_controller.rb

```diff
@@ -106,7 +106,7 @@ def list
     if sort_param == 'username'
       order_string = 'username ASC'
     elsif sort_param == 'last_activity'
-      order_string = 'last_updated DESC'
+      order_string = 'updated_at DESC'
```

Comment:
@Manasa2850 I am not too sure switching this to `updated_at` fixes the problem. The `updated_at` is related to the user details I think, and the `last_updated` is related to the user's activities. 

I could be wrong but that's something you want to confirm, when is the `last_updated` changed and what changes affect the `updated_at` in Rails, the `updated_at` is a default column that updates the timestamp when the user details changes in the other columns which might be unrelated to the changes detected by the `last_updated` which might be looking at when a user was last active on the site.

Reply:
@RuthNjeri I think you're right with respect to the `updated_at` column reflecting the timestamp when changes are made in other columns. 
I don't find a column called `last_updated` in the User table so I'm not really sure how that's working.

Reply:
@Manasa2850 they are fetching it in line 137 as an alias for the most recently updated node for that user :sweat_smile:

Reply:
Ohh I missed that. Thanks @Tlazypanda! :smiley:

Reply:
If you need some input or more problem solving please reach out in chat! Thanks everyone for working together on this one!